Library/Homebrew/language/python.rb

```diff
@@ -139,11 +139,22 @@ def virtualenv_create(venv_root, python = "python", formula = self)
         venv
       end
 
+      def needs_python?(python)
```

Comment:
It would be nice to add documentation to this method (to follow Tim's precedent when he introduced the `virtualenv` code) and as this method is only used internally, either make it private or at least document it as private via `@api private`.